views/welcome.hbs

```diff
@@ -24,18 +24,11 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <h4>Pick a Username:</h4>
+        <h4>Login with Facebook</h4>
       </div>
       <div class="modal-body">
-        <form>
-          <div class="form-group">
-            <label for="username">Username:</label>
-            <input type="text" name="username" id="username" class="form-control" />
-          </div>
-        </form>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-primary btn-lg">Submit</button>
+        <a href="https://www.facebook.com/v2.8/dialog/oauth?client_id=1829373514000205&redirect_uri=http://localhost:3005/fb-login&scope=public_profile,email" class="btn btn-block btn-primary btn-lg">
```

Comment:
yes and then facebook servers make a request to our server endpoints with code in the querystring